problems/objects.js

```diff
@@ -187,15 +289,69 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = (str) => {
+  let characterObject = {};
+  let strArray = str.toLowerCase();
+  strArray = strArray.split(" ").join("");
```

Comment:
instead of doing this splitting and joining we can check our char while iterating and continue if it's a " "